src/System.ComponentModel.EventBasedAsync/tests/AsyncCompletedEventArgsTests.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.ComponentModel.EventBasedAsync.Tests
+{
+    public class AsyncCompletedEventArgsTests : AsyncCompletedEventArgs
```

Comment:
because method `RaiseExceptionIfNecessary(...)` which we want to test (below) is protected